blocks-common/b-page/__csp/b-page__csp.deps.js

```diff
@@ -0,0 +1,3 @@
+({
+    mustDeps: {block: 'i-bem', elems: ['html']}
```

Comment:
Last time we thought it should work, we found bug in deps-old tech.
For now we just use `elems` for `i-bem` deps in all code base.